README.md

```diff
@@ -54,6 +54,13 @@ make test # all tests
 make test TEST="-k product" # single module test
 ```
 
+The tests are run against prerecorded server-responses.
+If you need to (re-)record some cassettes, delete the stale ones in `test/test_playbooks/fixtures` and record new ones with
```

Comment:
The recording mode chosen here 'once' will only record requests if the tape file is new. It will not add new requests to an existing file.
Given that we broke down the fixtures to one tape file per module call, i think it is safer to not accidentially adding new requests (and keep the old stale requests around).